og_access/og_access.test

```diff
@@ -345,6 +345,59 @@ class OgAccessTestCase extends DrupalWebTestCase {
 
     $this->assertFalse(node_access('view', $node, $user2), 'Group member can not view unpublished node.');
   }
+
+  /**
+   * Test unpublished node.
+   */
+  function testUnpublishedGroupAccess() {
```

Comment:
this test should move under `og.test`. `og_access.test` should test only functionality of OG access module.